lib/features/request/deposit_page.dart

```diff
@@ -95,25 +95,30 @@ class DepositPage extends HookConsumerWidget {
     String payinAmount,
     String payoutAmount,
     Currency? selectedCurrency,
-  ) =>
-      Padding(
-        padding: const EdgeInsets.symmetric(horizontal: Grid.side),
-        child: FilledButton(
-          onPressed: () {
-            Navigator.of(context).push(
-              MaterialPageRoute(
-                builder: (context) => PaymentDetailsPage(
-                  payinAmount: payinAmount,
-                  payoutAmount: payoutAmount,
-                  payinCurrency: selectedCurrency?.code.toString() ?? '',
-                  payoutCurrency: CurrencyCode.usdc.toString(),
-                  exchangeRate: selectedCurrency?.exchangeRate.toString() ?? '',
-                  transactionType: TransactionType.deposit,
-                ),
-              ),
-            );
-          },
-          child: Text(Loc.of(context).next),
+  ) {
+    final disabled = double.tryParse(payinAmount) == 0;
+
+    void onPressed() {
```

Comment:
nit: we started using [arrow syntax](https://dart.dev/language/functions) more in this repo, this would be a place to use it!

Reply:
also.. just noticed that `onPressed()` is always defined within another function. superrr nit but could we pull these out and just make them private functions (i.e. `_onPressed()` on the same level as `_buildNextButton()`)?

Reply:
@ethan-tbd would you h8 me if I left it as a nested function? Since `onPressed()` is only needed in the context of the specific `_buildNextButton()` or equivalent function, it feels nicely encapsulated and readable to me.

I can definitely update to arrow functions.

Reply:
nah you can leave it in if ya want! just brought it up since we don't do that anywhere else in the codebase